cantabular/queries.go

```diff
@@ -116,6 +116,30 @@ query($dataset: String!) {
 	}
 }`
 
+const QueryDimensionCategories = `
+query($dataset: String!, $variables: [String!]!) {
+    dataset(name: $dataset) {
+	variables(names: $variables) {
+	    edges {
+		node {
+		    name
+		    label
+		    categories {
+	    edges{
```

Comment:
json is mis-aligned. Be sure to show invisibles in your IDE and make sure you're consistent in using either tabs or spaces.